plugins/org.jkiss.dbeaver.ext.athena/plugin.xml

```diff
@@ -31,6 +31,7 @@
                         category="AWS"
                         webURL="https://docs.aws.amazon.com/athena/latest/ug/connect-with-jdbc.html"
                         propertiesURL="https://s3.amazonaws.com/athena-downloads/drivers/JDBC/SimbaAthenaJDBC-2.0.25.1001/doc/Simba+Athena+JDBC+Connector+Install+and+Configuration+Guide.pdf"
+                        driverDocumentationURL="Database-driver-Amazon-Athena"
```

Comment:
I would say it's not a driver, but database documentation.
Also it's not an url, but it's suffix :)

Reply:
databaseDocumentationSuffixURL?

Reply:
sounds better for me

Reply:
Ticket is already in Ready For QA, so maybe just left it as is